src/util-radix-tree.c

```diff
@@ -484,11 +482,13 @@ void SCRadixReleaseRadixTree(SCRadixTree *tree)
  *                   this key
  * \param netmask    The netmask (cidr) if we are adding an IP netblock; 255
  *                   if we are not adding an IP netblock
+ * \param exclusive  True (false) if the node should added if not present (present).
```

Comment:
I have a bit of trouble parsing this... what is `True (false)` and `present (present)`?

Reply:
I'll reword this and will repost.